cli/template/extras/src/server/api/trpc/base.ts

```diff
@@ -14,7 +14,18 @@
  *
  * These allow you to access things when processing a request, like the database, the session, etc.
  */
+
+/**
+ * 2. INITIALIZATION
+ *
+ * This is where the tRPC API is initialized, connecting the context and transformer. We also parse
+ * ZodErrors so that you get typesafety on the frontend if your procedure fails due to validation
+ * errors on the backend.
+ */
+import { initTRPC } from "@trpc/server";
 import { type CreateNextContextOptions } from "@trpc/server/adapters/next";
+import superjson from "superjson";
+import { ZodError } from "zod";
```

Comment:
same here - wish it didn't move these imports when they are clearly separated :/

we'll have to look over the comments cause those should not be moved